react/components/Table/Toolbar/index.js

```diff
@@ -0,0 +1,224 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+
+import { constants } from '../util'
+import SimpleInput from './SimpleInput'
+import ButtonDensity from './ButtonDensity'
+import ButtonFields from './ButtonFields'
+import ButtonNewLine from './ButtonNewLine'
+import ExtraActions from './ExtraActions'
+import ButtonToolbar from './ButtonToolbar'
+
+import IconUpload from '../../icon/Upload'
+import IconDownload from '../../icon/Download'
+import InputAutoComplete from './InputAutoComplete'
+
+const Container = ({ justify = 'end', children }) => (
+  <div id="toolbar" className={`mb5 flex flex-row w-100 justify-${justify}`}>
+    {children}
+  </div>
+)
+
+const BUTTON_TYPES = {
```

Comment:
Are those going to be used in props by the developer?

Reply:
Nope, just internally.